site/en/r2/tutorials/load_data/csv.ipynb

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
<p>reshuffle_each_iteration defaults to True. So you can remove it :)</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
Done